File: Misfit.API/misfit-web/src/app/models/userListingProperty.ts
export class UserListProperty {
  constructor(pageNo: number, pageSize: number ) {
    this.pageNumber = pageNo;
    this.pageSize = pageSize;
  }
    userName: string;
    startDate: string;
    endDate: string;
    sortBy: string;
    sortingOrder: string;
    pageNumber: number;
    pageSize: number;
  }

File: Misfit.CORE/ViewModels/UserListingProperty.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Misfit.CORE.ViewModels
{
    public class UserListingProperty : PagingBase
    {
        public string UserName { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public string SortBy { get; set; }
        public string SortingOrder { get; set; }
    }
}
